changes-set.txt

```diff
@@ -27,14 +27,36 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 4-Oct-22 brresi    brresi2
+ 4-Oct-22 opelresi  opelidres
+ 4-Oct-22 brresg    brresgOLD2
+ 4-Oct-22 opelresg  opelresgOLD2
+ 4-Oct-22 brres     brresOLD2
+ 4-Oct-22 brres     brresi
+ 4-Oct-22 opelres   opelresOLD2
+ 4-Oct-22 opelres   opelresi
+ 4-Oct-22 brresALTV brres       moved from PM's mathbox to main set.mm
+ 4-Oct-22 opelresALTV opelres   moved from PM's mathbox to main set.mm
+ 4-Oct-22 inxpssres [same]      moved from PM's mathbox to main set.mm
+ 4-Oct-22 biancomd  [same]      moved from PM's mathbox to main set.mm
+ 4-Oct-22 biancomi  biancom     moved from PM's mathbox to main set.mm
+ 4-Oct-22 dvreslem  [same]      commutation in theorem
  1-Oct-22 eu5       dfeu        this is now a rederivation
  1-Oct-22 mo2v      dfmo        this is now a rederivation
  1-Oct-22 mo2       mof         "f-version" of df-mo
 26-Sep-22 euequ1    euequ
 26-Sep-22 eueq1     eueqi       inference associated with eueq
+18-Sep-22 brinxp2ALTV brinxp2   moved from PM's mathbox to main set.mm
+18-Sep-22 brinxp2   brinxp2OLD
 17-Sep-22 zfnuleu   ---         deleted; use nulmo instead
 17-Sep-22 bm2.5ii   uniordint   more informative label
 17-Sep-22 bm1.1     ---         deleted; use axextmo instead
+14-Sep-22 idssxp    [same]      proved by pre-function definitions
+14-Sep-22 idssxp    idssxpOLD 
```

Comment:
idssxp is not changed in this PR!? Than this should be removed (in general, only changing a proof should not documented in this file...).

Reply:
done ( https://github.com/metamath/set.mm/pull/2873/commits/1961e2a3ae7faae6628f84557a3e4cf029cdc556 )